src/components/FishCard.js

```diff
@@ -1,40 +1,53 @@
+import { useState } from 'react';
 import styled from 'styled-components';
 
 export default function FishCard({ fish }) {
+  const [showDetails, setShowDetails] = useState(false);
   return (
     <FishContainer>
       <FishName>{fish.FishGerman}</FishName>
-      <PropertyList>
```

Comment:
When I do see it right, you kicked out the ul and you are starting directly with li, right? Therefore you should keep the Propertylist😊

Reply:
Oh, das habe ich übersehen, danke für den Hinweis.